content/guides/shimming.md

```diff
@@ -158,3 +159,86 @@ module.exports = {
 ## Node Built-Ins
 
 Node built-ins, like `process`, can be polyfilled right directly from your configuration file without the use of any special loaders or plugins. See the [node configuration page](/configuration/node) for more information and examples.
+
+
+## Loading polyfills on demand
+
+It's really common in web projects to include polyfills in the main bundle. This is not recommended since we are penalizing modern browsers users, since they will have to download a bigger file unnecessarely.
```

Comment:
"since" twice in the last sentence is a bit repetitive and "unnecessarely" is spelled wrong. Maybe:

> It's really common in web projects to include polyfills in the main bundle. This penalizes modern browser users as they will be forced to download a bigger file containing unneeded polyfills.